include/dsn/dist/failure_detector/fd.types.h

```diff
@@ -120,24 +108,7 @@ namespace dsn { namespace fd {
         bool allowed;
```

Comment:
Is "beacon_ack.is_master" field redundant? In my opinion,  "beacon_ack.this_node==beacon_ack.primary_node" implies that "this_node" is master?